drydock/templates/kustomized/tutor13/extensions/ingress.yml

```diff
@@ -43,9 +43,25 @@ spec:
     {%- for host in DRYDOCK_INGRESS_EXTRA_HOSTS %}
     - {{ host }}
     {%- endfor %}
+    {% if DRYDOCK_CUSTOM_CERTS -%}
+    secretName: {{ DRYDOCK_CUSTOM_CERTS["secret_name"]|default("custom-tls-certs") }}
+    {% else -%}
     secretName: {{ K8S_NAMESPACE }}-tls
+    {%- endif %}
   {%- endif %}
 {% if ENABLE_HTTPS -%}
+{% if DRYDOCK_CUSTOM_CERTS -%}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: {{ DRYDOCK_CUSTOM_CERTS["secret_name"]|default("custom-tls-certs") }}
```

Comment:
can you move the defaults to `defaults.yml`

Reply:
Not in this case because this value is inside the `DRYDOCK_CUSTOM_CERTS` which I want to default to an empty dict `{}`.

Reply:
right